lib/lob/resource_base.ex

```diff
@@ -40,7 +42,7 @@ defmodule Lob.ResourceBase do
       end
 
       @spec base_url :: String.t
-      defp base_url, do: "#{Application.get_env(:lob_elixir, :api_endpoint)}/#{unquote(endpoint)}"
+      defp base_url, do: "#{@api_url}/#{unquote(endpoint)}"
```

Comment:
Not sure if it matters or not, but would you ever want your library/tests to point at a different url? If so, you could OR the api_url and set a default if it's not defined.